HW06.md

```diff
@@ -0,0 +1,1131 @@
+    ## 95 percent confidence interval:
+    ##  -0.4892553  0.3113365
+    ## sample estimates:
+    ## mean in group H3K27_Mutated            mean in group WT 
+    ##                    1.098380                    1.187339 
+    ## 
+    ## 
+    ## [[24]]
+    ## 
+    ##  Welch Two Sample t-test
+    ## 
+    ## data:  MICA
+    ## t = -2.2467, df = 43.204, p-value = 0.02982
+    ## alternative hypothesis: true difference in means is not equal to 0
+    ## 95 percent confidence interval:
+    ##  -3.2774150 -0.1770228
+    ## sample estimates:
+    ## mean in group H3K27_Mutated            mean in group WT 
+    ##                   0.1675271                   1.8947460 
+    ## 
+    ## 
+    ## [[25]]
+    ## 
+    ##  Welch Two Sample t-test
+    ## 
+    ## data:  CDKN2A
+    ## t = -2.604, df = 308.33, p-value = 0.00966
+    ## alternative hypothesis: true difference in means is not equal to 0
+    ## 95 percent confidence interval:
+    ##  -1.034988 -0.144060
+    ## sample estimates:
+    ## mean in group H3K27_Mutated            mean in group WT 
+    ##                  0.06456416                  0.65408801 
+    ## 
+    ## 
+    ## [[26]]
+    ## 
+    ##  Welch Two Sample t-test
+    ## 
+    ## data:  EZH2
+    ## t = -1.0244, df = 441.11, p-value = 0.3062
+    ## alternative hypothesis: true difference in means is not equal to 0
+    ## 95 percent confidence interval:
+    ##  -0.23196548  0.07300789
+    ## sample estimates:
+    ## mean in group H3K27_Mutated            mean in group WT 
+    ##                   0.3972694                   0.4767482 
+    ## 
+    ## 
+    ## [[27]]
+    ## 
+    ##  Welch Two Sample t-test
+    ## 
+    ## data:  BRD2
+    ## t = -2.9219, df = 758.05, p-value = 0.003582
+    ## alternative hypothesis: true difference in means is not equal to 0
+    ## 95 percent confidence interval:
+    ##  -0.6737638 -0.1322481
+    ## sample estimates:
+    ## mean in group H3K27_Mutated            mean in group WT 
+    ##                   0.9597985                   1.3628045 
+    ## 
+    ## 
+    ## [[28]]
+    ## 
+    ##  Welch Two Sample t-test
+    ## 
+    ## data:  CDC42
+    ## t = -2.3486, df = 176.27, p-value = 0.01996
+    ## alternative hypothesis: true difference in means is not equal to 0
+    ## 95 percent confidence interval:
+    ##  -4.9703679 -0.4312928
+    ## sample estimates:
+    ## mean in group H3K27_Mutated            mean in group WT 
+    ##                    4.834198                    7.535028
+
+Now create a graphing function to create boxplots to visualize the
+results. Plot expression on the y-axis. The graph should look like [this
+example](IRX1_example.png)
+
+``` r
+#selecting columns of interest to make new dataframe
+trimmed_df <- GBM.genes.of.interest %>%  
+  dplyr::select(gene_symbol, type, TPM)
+
+#create graphing function
+graph_tpm <- function(df) {
+      plot <- ggplot(subset(trimmed_df, trimmed_df$gene_symbol == genes_of_interest[i]),
```

Comment:
I hit a block which didn't allow me to get to this point, so it is really insightful to see how you had to think ahead exactly how your graphing function would work within a for loop. This first line of code especially gets to the crux of a function made for a for loop, which is super interesting.